easypg/testsetup.go

```diff
@@ -95,6 +95,11 @@ func WithTestDB(m *testing.M, action func() int) int {
 		}
 	}
 
+	// check if a previous connection is still lingering
+	if _, err := os.Stat(filepath.Join(rootPath, ".testdb/run/pid")); err == nil {
+		stopDatabaseConnection(rootPath)
```

Comment:
This should not be a fatal error if it does not work (e.g. if the database was stopped by a reboot, but the PID file was not cleaned up). I suggest returning the error from this function and handling it with either `logg.Error()` or `logg.Fatal()`.

Reply:
Sound like a more graceful handling which I can just support to do. [2b8ea74](https://github.com/sapcc/go-bits/pull/193/commits/2b8ea7492fc0c85082ed5943aab277c7de1ef098)